core/trino-main/src/main/java/io/trino/sql/planner/LogicalPlanner.java

```diff
@@ -319,7 +319,7 @@ private RelationPlan createAnalyzePlan(Analysis analysis, Analyze analyzeStateme
                 idAllocator.getNextId(),
                 new AggregationNode(
                         idAllocator.getNextId(),
-                        TableScanNode.newInstance(idAllocator.getNextId(), targetTable, tableScanOutputs.build(), symbolToColumnHandle.build(), false),
+                        TableScanNode.newInstance(idAllocator.getNextId(), targetTable, tableScanOutputs.build(), symbolToColumnHandle.build(), false, Optional.empty()),
```

Comment:
`Introduce TableScanNode#useConnectorTablePartitioning` commit itself is a noop. It adds fields which are not used. Squash with next commit before final merge.

Reply:
That commit extracts mechanical changes to separate commit (we used to do that in the past too with other featuers). This way changes that are not trivial (e.g adding rule) are separated from mindless changes.